pkg/scalers/rabbitmq_scaler.go

```diff
@@ -41,6 +41,7 @@ const (
 	rabbitMetricType                       = "External"
 	rabbitRootVhostPath                    = "/%2F"
 	rmqTLSEnable                           = "enable"
+	connectionName                         = "keda"
```

Comment:
Maybe better if it would be like `keda-<namespace>-<something>`?

Reply:
I've checked and `ScalerConfig` already has the information about ScaledObject and namespace, so we could include it:
https://github.com/kedacore/keda/blob/0771b354cb294887f462ca227100f6836204ea2d/pkg/scalers/scalersconfig/scalersconfig.go#L30-L33

If you build the connection name during the scaler creation, you can access to the info

Reply:
I'll make this change shortly